Makefile

```diff
@@ -117,5 +119,13 @@ debug-build: build ## Build binary with debug info, symbols, and no optimization
 .PHONY: debug-container
 debug-container: debug-build container ## Build container with debug binary
 
+.PHONY: generate-manifests
+generate-manifests: ## Generate manifests using Helm.
+	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) $(HELM_TEMPLATE_EXTRA_ARGS) -n nginx-gateway | cat $(strip $(MANIFEST_DIR))/namespace.yaml - > $(strip $(MANIFEST_DIR))/nginx-gateway.yaml
```

Comment:
since `HELM_TEMPLATE_EXTRA_ARGS` only applies to generating all manifests combined, can we rename it to reflect that? For example `HELM_TEMPLATE_EXTRA_ARGS_FOR_ALL_MANIFESTS_FILE`